fixtures/devtools/standalone/index.html

```diff
@@ -251,63 +253,65 @@ <h1>List</h1>
                   }
                 }
               }
-            }}
-          />
-        );
-      }
+            }
+          }}
+        />
+      );
+    }
 
-      const set = new Set(['abc', 123]);
-      const map = new Map([['name', 'Brian'], ['food', 'sushi']]);
-      const setOfSets = new Set([new Set(['a', 'b', 'c']), new Set([1, 2, 3])]);
-      const mapOfMaps = new Map([['first', map], ['second', map]]);
-      const typedArray = Int8Array.from([100, -100, 0]);
-      const immutable = Immutable.fromJS({
-        a: [{ hello: 'there' }, 'fixed', true],
-        b: 123,
-        c: {
-          '1': 'xyz',
-          xyz: 1,
-        },
-      });
-
-      function UnserializableProps() {
-        return (
-          <ChildComponent
-            map={map}
-            set={set}
-            mapOfMaps={mapOfMaps}
-            setOfSets={setOfSets}
-            typedArray={typedArray}
-            immutable={immutable}
-          />
-        );
-      }
+    const set = new Set(['abc', 123]);
+    const map = new Map([['name', 'Brian'], ['food', 'sushi']]);
+    const setOfSets = new Set([new Set(['a', 'b', 'c']), new Set([1, 2, 3])]);
+    const mapOfMaps = new Map([['first', map], ['second', map]]);
+    const typedArray = Int8Array.from([100, -100, 0]);
+    const immutable = Immutable.fromJS({
+      a: [{ hello: 'there' }, 'fixed', true],
+      b: 123,
+      c: {
+        '1': 'xyz',
+        xyz: 1,
+      },
+    });
 
-      function ChildComponent(props: any) {
-        return null;
-      }
+    function UnserializableProps() {
+      return (
+        <ChildComponent
+          map={map}
+          set={set}
+          mapOfMaps={mapOfMaps}
+          setOfSets={setOfSets}
+          typedArray={typedArray}
+          immutable={immutable}
+        />
+      );
+    }
 
-      function InspectableElements() {
-        return (
-          <Fragment>
-            <SimpleValues />
-            <ObjectProps />
-            <UnserializableProps />
-            <CustomObject />
-          </Fragment>
-        );
-      }
+    function ChildComponent(props: any) {
+      return null;
+    }
 
-      function App() {
-        return (
-          <Fragment>
-            <List />
-            <InspectableElements />
-          </Fragment>
-        );
-      }
+    function InspectableElements() {
+      return (
+        <Fragment>
+          <SimpleValues />
+          <ObjectProps />
+          <UnserializableProps />
+          <CustomObject />
+        </Fragment>
+      );
+    }
+
+    function App() {
+      return (
+        <Fragment>
+          <List />
+          <InspectableElements />
+        </Fragment>
+      );
+    }
+
+    ReactDOM.render(<App />, document.getElementById("root"));
+  </script>
+</body>
 
-      ReactDOM.render(<App />, document.getElementById("root"));
-    </script>
-  </body>
```

Comment:
Why all of the whitespace changes to this fixtures file? Seems unrelated to the PR.

Reply:
Yeah, i skip that i shouldn't have committed.